abci/abci.go

```diff
@@ -13,3 +19,127 @@ func (v ABCIClientVersion) String() string {
 		"ABCIClientVersion2",
 	}[v]
 }
+
```

Comment:
nothing changed here, I just added a type assertion and put the functions that implement the `abci.Application` interface here.